core/common/base/src/main/java/org/eclipse/dataspaceconnector/core/DefaultCoreExtension.java

```diff
@@ -12,27 +12,36 @@
  *
  */
 
-package org.eclipse.dataspaceconnector.core.defaults;
+package org.eclipse.dataspaceconnector.core;
 
 import dev.failsafe.RetryPolicy;
 import okhttp3.EventListener;
 import okhttp3.OkHttpClient;
 import org.eclipse.dataspaceconnector.core.base.OkHttpClientFactory;
+import org.eclipse.dataspaceconnector.core.event.EventExecutorServiceContainer;
 import org.eclipse.dataspaceconnector.spi.EdcSetting;
+import org.eclipse.dataspaceconnector.spi.security.CertificateResolver;
+import org.eclipse.dataspaceconnector.spi.security.PrivateKeyResolver;
+import org.eclipse.dataspaceconnector.spi.security.Vault;
+import org.eclipse.dataspaceconnector.spi.system.ExecutorInstrumentation;
 import org.eclipse.dataspaceconnector.spi.system.Inject;
 import org.eclipse.dataspaceconnector.spi.system.Provider;
 import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
 import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.eclipse.dataspaceconnector.spi.system.vault.NoopCertificateResolver;
+import org.eclipse.dataspaceconnector.spi.system.vault.NoopPrivateKeyResolver;
+import org.eclipse.dataspaceconnector.spi.system.vault.NoopVault;
 import org.eclipse.dataspaceconnector.spi.transaction.NoopTransactionContext;
 import org.eclipse.dataspaceconnector.spi.transaction.TransactionContext;
 
 import java.time.temporal.ChronoUnit;
+import java.util.concurrent.Executors;
 
 /**
- * Provides defaults for various services.
- * Provider methods are only invoked if no other implementation was found on the classpath.
+ * Provides default service implementations for fallback
  */
-public class DefaultServicesExtension implements ServiceExtension {
+public class DefaultCoreExtension implements ServiceExtension {
```

Comment:
suggestion: how about naming these types of extensions `Default...ervicesExtension`, since they provide default core services...? Naming may get overly long at times though.

Reply:
@paullatzelsperger Actually that's a good advice, I renamed all these extensions in the `*DefaultServicesExtension` format and same for the `name()` method return value: `* Default Services`. Makes everything clearer.